### PR TITLE
Update import-certificate.md

### PR DESCRIPTION
**Pull request by sukihanabi**
#1171
**Action taken**

_Added an Example 3:_
Get-ChildItem –Path c:\import\export.cer | Import-Certificate –CertStoreLocation cert:\LocalMachine\My

**Steps taken to test the cmdlets:**

_1._) _Tried Example 1: works successfully_
$file = ( Get-ChildItem -Path C:\files\root.cer )
$file | Import-Certificate -CertStoreLocation cert:\CurrentUser\Root

Screenshot: 
https://snipboard.io/q0N72O.jpg

_2_.) Tried Example 2: works successfully__
PS C:\>Set-Location -Path cert:\CurrentUser\My
PS C:\>Import-Certificate -Filepath "C:\files\intermediate.cert"

Screenshot: 
https://snipboard.io/m3obQ7.jpg

__3.) Run command in Technet blog: https://blogs.technet.microsoft.com/scotts-it-_blog/2014/12/30/working-with-certificates-in-powershell/_

cmd: Get-ChildItem –Path c:\import\export.cer | Import-Certificate –CertStoreLocation cert:\LocalMachine\My

See screenshot:
https://snipboard.io/vUBS5h.jpg

_4.) Tried export and import from 1171_

Export-Certificate –Cert (Get-ChildItem –Path Cert:\LocalMachine\My\Thumbprint) –FilePath C:\Users\cert2.cer
Import-Certificate -Filepath c:\users\cert.cer -CertStoreLocation "cert:\LocalMachine\Root"

See screenshot:
https://snipboard.io/HsEr14.jpg - This worked, since I have an exported cert then imported the cert to the localmachine cert
https://snipboard.io/QtOs10.jpg - For this screenshot, I guess this will work if I have -cert in my ldap

_References :_
https://blogs.technet.microsoft.com/scotts-it-blog/2014/12/30/working-with-certificates-in-powershell/
https://docs.microsoft.com/en-us/powershell/module/pkiclient/export-certificate?view=win10-ps

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by e0i** [comment]
@sukihanabi Please refer the issue that this PR addresses. Thanks.

File: docset/windows/pkiclient/import-certificate.md [hunk 1]
```diff
@@ -52,6 +52,13 @@ PS C:\>Import-Certificate -Filepath "C:\files\intermediate.cert"
 
 This example imports the certificate from the file into the current store.
 
+### EXAMPLE 3
+```
+PS C:\>Get-ChildItem –Path c:\import\export.cer | Import-Certificate –CertStoreLocation cert:\LocalMachine\My
+
+```
+In this example Get-ChildItem locates the certificate and sends the result to the second cmdlet (Import-Certificate). Import-Certificate specifies which store the certificate should be imported.
```
Comment (JohanFreelancer9) [inline docset/windows/pkiclient/import-certificate.md]:
```suggestion
In this example Get-ChildItem locates the certificate and sends the result to the second cmdlet (Import-Certificate). Import-Certificate specifies to which store the certificate should be imported to.
```

**Comment by get-itips** [comment]
OK

**Comment by tseward** [comment]
Example 1 and 3 are functionally identical. I don't think this is needed.

**Comment by sukihanabi** [comment]
@tseward , agreed! It's a different way the example 1 will work. I guess what the issuer hasn't have is, an exported certificate before he import the cert . But if he has he will be able to import it with the given example.

**Comment by e0i** [comment]
@sukihanabi Which issue does this PR refers to?

**Comment by sukihanabi** [comment]
> @sukihanabi Which issue the PR refers to?
@e0i
Apologies, I missed to add it. It's #1171

**Comment by e0i** [comment]
Closing as no changes have been found to be necessary

**Pull request closed without merging**